docs/fieldnorm/life_cycle.md

```diff
@@ -1,7 +1,56 @@
+
+  d.       Establish a well-defined development and release cycle with set time periods for preparation, development, testing & releasing. This allows multiple developers to work cohesively on the project and creates structured testing and release moments during the implementation and maintenance processes. It also enables ample opportunities for user reflections on the deliverables during the lifetime of project.
+
+5.      Testing
+Before implementation, software should be extensively tested, verified and validated. While ISO 15189 does not provide specific guidelines on how to conduct software validation, it emphasizes the importance of ensuring that software is validated and verified following recognized industry standards. We advise the use of the software validation and verification procedures as described in this guideline.
+
+6.      Implementation
+Once everything is ready, a clear time point should be determined to release the software. The exact time point at which a software is released can be determined by the Development & Release cycle or released in an independent matter. The software functionality and release date should be clearly communicated with the end users.
+
+7.      Maintenance
+Maintenance is primarily centers around fixing bugs, monitoring the software, introducing new features and upgrading to newer versions with new end-users requirements . There should be a clear guideline for how new features, bug fixes and new versions are implemented into the software, preferably using the Development & Release cycle. A systematic approach will streamline the maintenance process and improve overall software quality and reliability.
+
+Development & Release cycle
+Bioinformaticians are advised to have a clearly determined Development & Release cycle procedure with set time periods for preparation, development and testing & releasing of features. Having a Development & Release cycle allows multiple developers to work on the same project in an organized manner. It also creates a platform for creating structural testing and release moments for the implementation and maintenance SDLC process steps. An example of a release cycle would be 1 week of preparation, 3 weeks of development followed by 1 week of Testing & releasing which overlaps with the first week (preparation) of the next cycle, resulting in a development & release cycle of 4 weeks.
+
+Preparation. During preparation goals of the release cycle are set. Within this phase requirements should become clear, and converted into tasks. These tasks are tracked in a task management system and are linked to the development cycle and resources are allocated to them. The goal of the cycle should be to finish and release all tasks that are determined in this phase.
```

Comment:
I feel like we're repeating the steps above

Reply:
I have removed the Development and release cycle example